go-controller/pkg/podannotation/pod_annotation.go

```diff
@@ -1,10 +1,11 @@
-package util
+package podannotation
```

Comment:
all our annotations like node_annotations, namespace_annotations are in util. What is the circular dependency that made you do this second commit?